### ISSUE TITLE
VMware Tools 10.3.x installation fails on Windows 8.1 and Windows Server 2012 R2 because of missing requirements

### ISSUE THREAD

**Issue by icnocop** [boxcutter/windows#185]
Hi.

The VMware Tools 10.3.x installation fails on Windows 8.1 and Windows Server 2012 R2 because of missing requirements.

VMware Tools 10.3.x requires Microsoft Windows update [KB2919355](http://support.microsoft.com/kb/2919355) on Windows 8.1 and Windows Server 2012 R2.

See https://kb.vmware.com/s/article/55798.

Otherwise the following appears in the packer log:
```
WARNING: Error 1603 was returned by: "d:\setup.exe" /S ...
```

The VMware Tools setup log contains the following error:
```
ERROR: OS compatibility requires update KB2919355
```

builder type: vmware-iso
VMware ESXi 6.7.0

**Comment by arizvisa** [comment]
@icnocop, I assume that also means win2016-standard-*.json, right? Does this issue also include win2008r2-*.json?

**Comment by arizvisa** [comment]
The reason I mention it is because the KB article includes Win7, but VMware Tools appears to be installed on all my W7 VMs, so just wondering how far back I'd need to go.

**Comment by icnocop** [comment]
The VMware KB article mentions that Windows 7 must have SP1 installed in order to install VMware Tools.
Are you running at least Windows 7 SP1?

The same applies for:
- Windows Server 2008 R2, SP1 is required.
- Windows Server 2008, SP2 is required.
- Windows Vista, SP2 is required.

The Microsoft Windows update KB2919355 is required only for Window 8.1 and Windows Server 2012 R2.

**Comment by arizvisa** [comment]
I think this issue is closed by PR #243

**Comment by arizvisa** [comment]
It it doesn't, let me know and I'll see what I can do.
